### PR TITLE
Feat/add catboy token

### PR DESCRIPTION
**Pull request by Catboyteam**

### CODE REVIEW

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 1]
```diff
@@ -3392,6 +3392,38 @@
...
       ]
+    },
+    "0x4373f265c4d680dec5635152187b505f3820570ad4d8bbf811ff93ae826323ed": {
+      "tokenId": "0x4373f265c4d680dec5635152187b505f3820570ad4d8bbf811ff93ae826323ed",
+      "deployer": "0xcE6084118756Da652F793fd52b45D6FCd48b52D8",
+      "originalMinter": null,
+      "prettySymbol": "CATBOY",
+      "decimals": 18,
+      "originAxelarChainId": "ethereum",
+      "tokenType": "canonical",
```
Comment (Olanetsoft) [inline registry/mainnet/interchain/squid.tokenlist.json:3403]: is this "canonical" or "customInterchain"?

If you deployed token managers programmatically yourself to link the token as an interchain token, then it is "customInterchain". Only if you deploy it using the portal as an existing token, that's when it is "canonical." Please update.

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 2]
```diff
@@ -3392,6 +3392,38 @@
...
+      },
+      "deploymentMessageId": "",
+      "chains": [
+        {
+          "symbol": "CATBOY",
+          "name": "Catboy",
+          "axelarChainId": "ethereum",
+          "tokenAddress": "0xcA9b8D6df0729D85DCfc8eF8Bb18af1aD1990786",
+          "tokenManager": "0x09DB780F75f73C6a580D21d4ab5D71bAF860c2D2",
+          "tokenManagerType": "mintBurn"
```
Comment (Olanetsoft) [inline registry/mainnet/interchain/squid.tokenlist.json:3416]: Can you double-check that this is the type for both tokens?

**Review by Olanetsoft — CHANGES_REQUESTED** [review]

File: registry/mainnet/interchain/squid.tokenlist.json (hunk 1 above)
Comment (Catboyteam) [reply registry/mainnet/interchain/squid.tokenlist.json]: Corrected

File: registry/mainnet/interchain/squid.tokenlist.json (hunk 2 above)
Comment (Catboyteam) [reply registry/mainnet/interchain/squid.tokenlist.json:3416]: Both are mint burn

**Review by Olanetsoft — APPROVED** [review]

**Review by Olanetsoft — APPROVED** [review]

**Pull request merged**